src/NinjaTools.FluentMockServer.API/Logging/LoggingInitializer.cs

```diff
@@ -0,0 +1,51 @@
+using System.IO;
+using System.IO.Abstractions;
+using System.Threading.Tasks;
+using NinjaTools.FluentMockServer.API.Types;
+
+namespace NinjaTools.FluentMockServer.API.Logging
+{
+    public static class MockServerPaths
+    {
+        public const string LogRoot = "/var/log/mock-server";
+
+        public static class Logs
+        {
+            public static readonly string Setups = Path.Combine(LogRoot, "setups");
+            public static readonly string Requests = Path.Combine(LogRoot, "requests");
+        }
+
+        public static string Configs  = "/etc/mock-server/config/";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'Configs' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)